code/modules/projectiles/guns/misc/syringe_gun.dm

```diff
@@ -61,6 +61,10 @@
 
 /obj/item/gun/syringe/attackby(obj/item/A, mob/user, params, show_msg = TRUE)
 	if(istype(A, /obj/item/reagent_containers/syringe))
+		var/obj/item/reagent_containers/syringe/S = A //to check shootability //monkestation edit
```

Comment:
More descriptive variable names would be appreciated

Reply:
You're right.  When you're staring at a file and immersed in variables like A there, written without being expected to be used more than one line lower, you kinda lose sight easily lol.